Cargo.toml

```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 description = "Create and decode JWTs in a strongly typed way."
 homepage = "https://github.com/Keats/jsonwebtoken"
 repository = "https://github.com/Keats/jsonwebtoken"
-keywords = ["jwt", "web", "api", "token", "jwk"]
+keywords = ["jwt", "web", "api", "token", "jwk", "jwks"]
```

Comment:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-keywords-field

> Note: crates.io has a maximum of 5 keywords. Each keyword must be ASCII text, start with a letter, and only contain letters, numbers, _ or -, and have at most 20 characters.

Reply:
I keep forgetting about that :(